app/presenters/candidate_interface/application_form_presenter.rb

```diff
@@ -177,10 +177,10 @@ def references_selection_path
       end
     end
 
-    def work_experience_path
+    def work_experience_path(params = nil)
       if application_form.feature_restructured_work_history
         if application_form.application_work_experiences.any? || application_form.work_history_explanation.present?
-          Rails.application.routes.url_helpers.candidate_interface_restructured_work_history_review_path('return-to' => 'application-review')
+          Rails.application.routes.url_helpers.candidate_interface_restructured_work_history_review_path(params)
```

Comment:
It's a bit surprising to pass params to `work_experience_path` and it not have an effect for all the possible branches of the `if` statement, so I'd add `params` to all the `_path` calls in this method?

Hopefully we can clean up all the non-`restructured` work history in the near future

Reply:
🤦 thanks